Localization/en-US/home.po

```diff
@@ -0,0 +1,100 @@
+msgid ""
+msgstr ""
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20200705\n"
+"Target-Language: \n"
+"Target-Version: \n"
+"Target-Last-Modified: \n"
+
+msgid "I AM"
+msgstr ""
+
+msgid "One"
+msgstr ""
+
+msgid "IN A MILLION"
+msgstr ""
+
+msgid "Regardless if you are already folding or haven't heard a word about it before, we need your help to reach our goal – which is 1 million folders."
+msgstr ""
+
+msgid "Start folding now"
+msgstr ""
+
+msgid "Hide this and watch the video."
+msgstr ""
+
+msgid "START FOLDING NOW"
+msgstr ""
+
+msgid "Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use."
+msgstr ""
+
+msgid "Available for:"
+msgstr ""
+
+msgid "Linux"
+msgstr ""
+
+msgid "Windows"
+msgstr ""
+
+msgid "Mac"
+msgstr ""
+
+msgid "Start folding"
+msgstr ""
+
+msgid "JOIN THE COMMUNITY"
+msgstr ""
+
+msgid "The Folding@Home community is spread over countless forums, websites and social media."
+msgstr ""
+
+msgid "Here's a list of our official channels."
+msgstr ""
+
+msgid "For tech support:"
+msgstr ""
+
+msgid "foldingforum.org"
+msgstr ""
+
+msgid "[Facebook](https://www.facebook.com/Foldinghome-136059519794607/)"
+msgstr ""
+
+msgid "[Twitter](https://twitter.com/foldingathome)"
+msgstr ""
+
+msgid "1 in a million"
+msgstr ""
+
+msgid "WHY WE NEED YOU"
+msgstr ""
+
+msgid "Folding@home is a project focused on disease research. The problems we are solving require so many computer calculations and we need your help to find the cures!"
+msgstr ""
+
+msgid "Fight diseases Together with us."
+msgstr ""
+
+msgid "The Folding@home software allows you to share your unused computer power with us ? so that we can research even more potential cures."
```

Comment:
```suggestion
msgid "The Folding@home software allows you to share your unused computer power with us – so that we can research even more potential cures."
```